Dockerfile

```diff
@@ -1,9 +1,6 @@
-FROM rust:latest as builder
-WORKDIR /usr/src
-RUN rustup target add x86_64-unknown-linux-musl
+FROM messense/rust-musl-cross:x86_64-musl as builder
```

Comment:
It seems that this file should be rebased after #456

Reply:
Already has done 🙂